book/visual-effects.md

```diff
@@ -1332,7 +1332,8 @@ the [`mix-blend-mode` property][mix-blend-mode-def], like this:
 </div>
 ```
 
-This HTML will look like Figure 5.
+This HTML will look like Figure 5.^[See the `browser.engineering`
```

Comment:
Make this paragraph print-only and retain the current one for web-only.

Reply:
Ok

Reply:
Done